### PR TITLE
Minor tests improvement

### PR DESCRIPTION
**Pull request by mvorisek**
submitting as a separate PR as requested

### CODE REVIEW

**Comment by derrabus** [comment]
What's the problem with those `setUp()` methods? I don't see this change as an improvement, tbh.

**Comment by mvorisek** [comment]
We instanciate multiple formatters with multiple configs. I do not see a reason why to store one specific formatter with one specific config but not all - this PR is therefore an unification and simplification IMO.

**Comment by mvorisek** [comment]
@derrabus does it make sense to you, can I get your approval here, please?

**Comment by derrabus** [comment]
Again, I don't see the improvement. The tests are fine, I wouldn't change them.

**Comment by derrabus** [comment]
Let's close this PR. Thank you for your proposal though.

File: tests/SqlFormatterTest.php [hunk 1]
```diff
@@ -25,65 +24,63 @@
...
+        $this->assertSame($html, $formatter->highlight($sql));
     }
 
     public function testHighlightBinary(): void
     {
-        $sql = 'SELECT "' . pack('H*', 'ed180e98a47a45b3bdd304b798bc5797') . '" AS BINARY';
+        $highlighter = new HtmlHighlighter();
+        $formatter   = new SqlFormatter($highlighter);
 
-        if (defined('ENT_IGNORE')) {
```
Comment (mvorisek) [inline tests/SqlFormatterTest.php:61]: @derrabus can you please reopen so I can amend this PR and propose this change only - I belive the l61 condition is definitely not needed

Comment (derrabus) [reply tests/SqlFormatterTest.php:61]: You're right. Please submit this change as a new PR.

Comment (mvorisek) [reply tests/SqlFormatterTest.php:61]: Can we please recykle this PR? I will amend/squash.

**Pull request closed without merging**